utils/interface.py

```diff
@@ -0,0 +1,528 @@
+        ) as omnisci_server_launched:
+
+            print("DOWNLOADING DATA")
+            dataset_download_cmdline = ["python3", ibis_data_script, "download"]
+            conda_env.run(dataset_download_cmdline)
+
+            print("IMPORTING DATA BY OMNISCI")
+            dataset_import_cmdline = [
+                "python3",
+                ibis_data_script,
+                "omniscidb",
+                "-P",
+                str(omnisci_server_launched.server_port),
+                "--database",
+                args.database_name,
+            ]
+            conda_env.run(dataset_import_cmdline)
+
+            print("RUNNING TESTS")
+            ibis_tests_cmdline = [
+                "pytest",
+                "-m",
+                "omniscidb",
+                "--disable-pytest-warnings",
+                "-k",
+                args.expression,
+                f"--html={report_file_path}",
+            ]
+
+            os.environ["IBIS_TEST_OMNISCIDB_DATABASE"] = args.database_name
+            os.environ["IBIS_TEST_DATA_DB"] = args.database_name
+            os.environ["IBIS_TEST_OMNISCIDB_PORT"] = str(omnisci_server_launched.server_port)
+            # pytest depends on above env variables
+            conda_env.run(ibis_tests_cmdline, cwd=args.ibis_path)
+
+
+def benchmark(args):
+    with create_conda_env(
+        args.ibis_path,
+        args.env_name,
+        args.env_check,
+        args.env_save,
+        args.python_version,
+        args.ci_requirements,
+    ) as conda_env:
+
+        benchmark_script_path = os.path.join(args.omniscript_path, "run_ibis_benchmark.py")
+        benchmark_cmd = ["python3", benchmark_script_path]
+
+        possible_benchmark_args = [
+            "bench_name",
+            "data_file",
+            "dfiles_num",
+            "iterations",
+            "dnd",
+            "dni",
+            "validation",
+            "optimizer",
+            "no_ibis",
+            "no_pandas",
+            "pandas_mode",
+            "ray_tmpdir",
+            "ray_memory",
+            "no_ml",
+            "gpu_memory",
+            "db_server",
+            "db_port",
+            "db_user",
+            "db_pass",
+            "db_name",
+            "db_table_etl",
+            "db_table_ml",
+            "executable",
+            "omnisci_cwd",
+            "port",
+            "http_port",
+            "calcite_port",
+            "user",
+            "password",
+            "ipc_conn",
+            "database_name",
+            "table",
+            "commit_omnisci",
+            "commit_ibis",
+            "import_mode",
+            "debug_timer",
+            "columnar_output",
+            "lazy_fetch",
+            "multifrag_rs",
+            "fragments_size",
+            "omnisci_run_kwargs",
+        ]
+        args_dict = vars(args)
+        for arg_name in list(args.interface._option_string_actions.keys()):
+            try:
+                pure_arg = re.sub(r"^--*", "", arg_name)
```

Comment:
```suggestion
                pure_arg = arg_name.lstrip('-')
```
let's not use `re` until we must :D